internal/cmd/genop/genop.go

```diff
@@ -196,6 +196,16 @@ func {{$name}}Const(n *node) {
 		{{- if $op.Shift}}
 		v := constant.Shift(vConstantValue(v0), token.{{tokenFromName $name}}, uint(vUint(v1)))
 		n.rval.Set(reflect.ValueOf(v))
+		{{- else if (eq $op.Name "/")}}
+		// TODO(mpl): exclude uints?
```

Comment:
As the default type for untyped integer constants it `int`, nothing needs to be done with regard to uints. Your comment can be disgarded.